internals/review_models.py

```diff
@@ -55,6 +54,33 @@ def get_raw_owner_file(cls, url):
     return owners_file.raw_content
 
 
+class GateDef(ndb.Model):
+  """Configuration for a review gate."""
+  gate_type = ndb.IntegerProperty(required=True)
+  approvers = ndb.StringProperty(repeated=True)
+
+  # TODO(jrobbins): Use these and phase out approval_devs.ApprovalFieldDef.
+  name = ndb.StringProperty()
+  description = ndb.StringProperty()
+  rule = ndb.StringProperty()
+  team_name = ndb.StringProperty()
+  slo_initial_response = ndb.IntegerProperty()
+
+  @classmethod
+  def get_gate_def(cls, gate_type: int) -> GateDef:
+    """Load an existing GateDef and or create a new one."""
+    query: ndb.Query = GateDef.query(GateDef.gate_type == gate_type)
+    gate_defs = query.fetch(1)
+    if gate_defs:
+      gate_def = gate_defs[0]
+    else:
+      logging.info(f'Creating empty GateDef for {gate_type}')
+      gate_def = GateDef(gate_type=gate_type)
+      gate_def.put()
+
+    return gate_def
```

Comment:
I remember last year trying to move away from adding these class methods directly on the model and instead putting them in a `_helpers.py` file. However, I'm blanking on why exactly that was. I think it had something to do with the possibility of circular imports. Either way, I don't think this is a problem now, so it's fine here. 🙂